gulpfile.js

```diff
@@ -188,4 +188,19 @@ gulp.task("bundle", async function () {
 	}
 });
 
-gulp.task("default", gulp.parallel("css", "bundle", "html", "md"));
+gulp.task("mjs", function () {
+	return gulp
+		.src(["./dist/color.esm.js*"])
+		.pipe(
+			rename((path) => {
+				if (path.extname === ".js") {
+					path.extname = ".mjs";
+				} else {
+					path.basename = "color.esm.mjs";
+				}
+			})
+		)
+		.pipe(gulp.dest("./dist/"));
+});
+
+gulp.task("default", gulp.parallel("css", "bundle", "mjs", "html", "md"));
```

Comment:
I might be wrong, but having `bundle` and `mjs` in parallel might result in an older version being renamed before rollup is done creating the new one.